client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -68,7 +68,7 @@ const InvestigatedPersonInfo = (props: Props) => {
     }, [investigationStatus.subStatus]);
 
     const updatedSubStatuses = useMemo(() =>
-        investigationStatus.mainStatus === InvestigationMainStatus.IN_PROCESS ? subStatuses.concat(noReason) : subStatuses,
+        investigationStatus.mainStatus === InvestigationMainStatus.IN_PROCESS ? subStatuses.concat(inProcess) : subStatuses,
```

Comment:
The value that is sent to the DB still needs to be no reason

Reply:
![image](https://user-images.githubusercontent.com/52863381/99652368-697da100-2a60-11eb-862e-c1348c70e125.png)

Reply:
The value that is sent to the DB is null not no reason